Src/Support/Google.Apis.Auth.AspNetCore/GoogleScopedPolicyProvider.cs

```diff
@@ -36,6 +36,10 @@ public GoogleScopedPolicyProvider(GoogleAuthenticationSchemeProvider scheme, IOp
 
         public Task<AuthorizationPolicy> GetDefaultPolicyAsync() => _default.GetDefaultPolicyAsync();
 
+        // This is only required in the ASP.NET Core v3 IAuthorizationPolicyProvider,
```

Comment:
Is there any *benefit* though? Does it perhaps give the impression of making this package work with v3, but without *actually* making it work?

Reply:
Now conditional, and returns the default fallback policy.